src/test/java/cms/gov/madie/measure/resources/MeasureTransferControllerTest.java

```diff
@@ -438,6 +447,72 @@ public void verifyIfOrganizationNamesAreUpdated() {
         "Innovaccer Url", persistedMeasure.getMeasureMetaData().getDevelopers().get(2).getUrl());
   }
 
+  @Test
+  public void testCreateMeasureDuplicateMeasureExceptionForQiCore() {
+    when(measureService.findAllByMeasureSetId(anyString()))
+        .thenReturn(List.of(Measure.builder().id("testMeasureId").build()));
+
+    assertThrows(
+        DuplicateMeasureException.class,
+        () -> controller.createMeasure(request, measure, LAMBDA_TEST_API_KEY));
+  }
+
+  @Test
+  public void testCreateMeasureNoOverwrittenNoDeleteForQiCore() {
+    when(measureService.findAllByMeasureSetId(anyString())).thenReturn(List.of());
```

Comment:
```suggestion
    when(measureService.findAllByMeasureSetId(anyString())).thenReturn(Collections.emptyList());
```
Prefer explicit return of emptyList to improve readability.